src/components/LikerComparisonCard.vue

```diff
@@ -85,6 +54,48 @@ export default {
       default: true,
     },
   },
+  computed: {
+    featureList() {
+      return [
+        LIKER_FEATURE.DISTRIBUTE_FUND,
+        LIKER_FEATURE.DISTRIBUTE_OWN_FEE,
+        LIKER_FEATURE.DETAILED_REPORT,
+        LIKER_FEATURE.PARTICIPATE_IN_COMMUNITY,
+        LIKER_FEATURE.BONUS_CONTENT,
+        LIKER_FEATURE.OFFLINE_EVENTS,
+      ];
+    },
+    plan() {
+      switch (this.type) {
+        case 'civic':
+          return {
+            price: 'USD5.00',
+            billingCycle: 'month',
+            features: this.featureList,
+          };
+
+        case 'general':
+        default:
+          return {
+            price: 'USD0.00',
+            billingCycle: 'month',
+            features: [LIKER_FEATURE.DISTRIBUTE_FUND],
+          };
+      }
+    },
+  },
+  methods: {
+    getFeatureAvailability(feature) {
+      return this.plan.features.includes(feature);
+    },
+    getFeatureText(feature) {
+      const i18nKey = `LikerFeature.${feature}`;
+      if (this.$t(i18nKey, this.$i18n.fallbackLocale) instanceof Object) {
+        return this.$t(`${i18nKey}.${this.type}`);
```

Comment:
should use `$te()` to check for key exist

Reply:
Updated